CSS/css_completions.py

```diff
@@ -1,9 +1,11 @@
-import re
 import sublime
 import sublime_plugin
+
+import re
 import timeit
 
 from functools import cached_property, wraps
+from typing import List, Optional, Tuple
```

Comment:
Same as in the other PR, importing List, Optional and Tuple is generally not necessary in 3.10+.